frost-ristretto255/src/lib.rs

```diff
@@ -11,11 +11,15 @@ use curve25519_dalek::{
 use rand_core::{CryptoRng, RngCore};
 use sha2::{Digest, Sha512};
 
-use frost_core::{frost, Ciphersuite, Field, FieldError, Group, GroupError};
+pub use frost_core::frost;
```

Comment:
This uses `pub` but others don't, was this intended?

Reply:
Whoops, fixed in cf6b639